website/docs/reference/widgets/chart.mdx

```diff
@@ -0,0 +1,412 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Chart
+
+This document explains how to effectively use the Chart widget to visualize and represent data.
+
+
+:::info
+Appsmith is integrated with [**FusionCharts**](https://www.fusioncharts.com) and has acquired a **re-distribution license**. With this license, you can use FusionCharts on the Appsmith **cloud** and **on-premise** platform. The use of the license is permitted as long as what you are building on Appsmith isn't used to compete with FusionCharts.<br/>
+
+Please be aware that the license **doesn't** include permission to use **Fusion Maps**. If you want to use this product, you may need to obtain a separate license.
+:::
+
+<VideoEmbed host="youtube" videoId="Rvlwb5123mE" title="How to use Chart Widget" caption="Configure Custom Chart"/>
+
+
+## Display data
+
+Appsmith offers the ability to display both static and dynamic data in a chart. The data should be provided in a JSON format:
+
+```js
+[
+  {
+    "x": "Product1",
+    "y": 20000
+  },..
+]
+```
+
+In this format, **`x`** denotes the **label** and **`y`** denotes the **value**. It's important to provide a value when displaying data to ensure that the chart can render correctly. Please note that the JSON format provided is only suitable for basic charts
+
+Appsmith offers several quick and easy ways to represent your data. With the Chart Type property, you can select one of the following types to visualize your data:
+* Line Charts, 
+* Bar Charts, 
+* Pie Charts, 
+* Column Charts, 
+* Area Charts, and  
+* [Custom Charts](#custom-charts)
+
+
+### Display static data
+
+To display static data, you can enter the desired data directly into the `Series Data property`, and it remains the same until manually updated or edited. This method is useful when you need to display fixed information, such as trends or comparisons, that don't change based on user interactions. 
+
+```js
+[
+  {
+    "x": "2021",
+    "y": 200000
+  },
+  {
+    "x": "2022",
+    "y": 320000
+  }
+]
+```
+
+### Display data dynamically
+
+You can display dynamic data by fetching it from APIs or databases. This can be done by using binding values to **Series Data property**. If you need to transform the incoming data to match this format, you can use **JSObject** to perform the transformation before binding it to the chart widget.
+
+For example, lets say you have a PostgreSQL database containing information about assets in your organization, such as their types and the date they were purchased. You can create a query that groups the assets by type and counts how many there are of each type, like this:
+
+```sql
```

Comment:
this example shows the transformation done in the query itself.  

so move the JS line below this example and give an example of the mustache syntax